### PR TITLE
Improve UI

### PR DESCRIPTION
**Pull request by lumapools**
Just improving the UI:

1. User Profile View Self (Left: Offers, Right: Requests)
Webp net-resizeimage Webp net-resizeimage (1)
2. List of Items and Detailed Item View
Webp net-resizeimage (2) Webp net-resizeimage (7)

3. User Profile View Other and Chat
Webp net-resizeimage (4) Webp net-resizeimage (5)

### CODE REVIEW

**Review by v1ruso — APPROVED** [review]
Looks sick!

File: app/src/main/res/values/themes.xml [hunk 1]
```diff
@@ -1,17 +1,16 @@
 <resources xmlns:tools="http://schemas.android.com/tools">
     <!-- Base application theme. -->
-    <style name="Theme.Sharingang" parent="Theme.MaterialComponents.DayNight.NoActionBar">
+    <style name="Theme.Sharingang" parent="Theme.Design.NoActionBar">
```
Comment (betrisey) [inline app/src/main/res/values/themes.xml:3]: Could we keep the Material design theme and change the colors?
The buttons are a bit bigger and there is an animation when clicking on it

**Review by betrisey — COMMENTED** [review]
It looks nice :)
I've just found a few details to fix:

Change the color of the icons
Screenshot 2021-05-15 at 19 54 03

- Update the color of the placeholder image
- The rating buttons are displayed on items that haven't been sold
- The color of the share button doesn't match the others
Screenshot 2021-05-15 at 19 56 12

When enabling the dark mode, the app becomes brighter haha (maybe just delete the dark theme if we have a dark theme by default)
Screenshot 2021-05-15 at 20 02 49

**Review by betrisey — APPROVED** [review]
Very good!

**Pull request merged**